ironmon_tracker/Program.lua

```diff
@@ -437,6 +439,28 @@ function Program.isInCatchingTutorial()
 	return Program.inCatchingTutorial
 end
 
+function Program.isInEvolutionScene()
+	local taskLocation = GameSettings.sEvoStructPtr
+	--print (taskLocation)
+	if GameSettings.versioncolor ~= "Ruby" and GameSettings.versioncolor ~= "Sapphire" and GameSettings.versioncolor ~= "" and GameSettings.versioncolor ~= nil then
```

Comment:
You can simplify this check by using the `GameSettings.game`, which has a value of `1` if the game is exactly Ruby or Sapphire, and some other number for the other games. You can also safely assume the values won't be `nil` or empty here, since those are required to be set before the Tracker will even start up.

So I'd recommend changing this to: `if GameSettings.game ~= 1 then`

Reply:
And for clarity for the reader, I'd also invert the comment directly below this to be explain what we're looking at, since a `~= 1` check doesn't tell us much at a glance.